FogCreek/Glitch-Community
# Cassey-community: fix img HTML issues in magikarp on no js page & parsing errors on a project page

## Links
* https://cassey-community.glitch.me (see how-to-test section)
* https://glitch.manuscript.com/f/cases/3328751/A11y-issues-on-magikarp-on-you-need-JS-fallback-page
* https://glitch.manuscript.com/f/cases/3328753/ 

## Changes:
* Width is automatically in pixels, so valid values for the 'width' attribute are numbers only. 
* Add alt=''

## How To Test:
For magikarp: 
*  Go to [cassey-community in the W3 HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcassey-community.glitch.me#l243c984) and you shouldn't see any warnings, compared to [glitch.com in the W3 HTML validator](https://validator.w3.org/nu/?doc=https%3A%2F%2Fglitch.com#l243c984) where you do.

For other changes: 
![image](https://user-images.githubusercontent.com/4480480/59642378-5c9da500-912a-11e9-95cc-c92c6043be6c.png)
- [HTML Checker bookmarklets can be found at this link under extras](https://validator.w3.org/nu/about.html)
- Make sure no visual regressions happened due to these changes


Caveats: 
 - figuring out how to remove the improper nesting for tooltips proved challenging; going to do that in a later PR. Also, the allowvr attribute isn't part of the spec yet but we need it for displaying embeds with VR content. You should see these two errors (and only these two) when you run the html checker on cassey-community.glitch.me/~one-of-your-projects. 
![image](https://user-images.githubusercontent.com/4480480/59642449-a5edf480-912a-11e9-8961-d4e520a137af.png)
